tests/swagger_generation/registries/include_hidden_api_authenticator.py

```diff
@@ -0,0 +1,448 @@
+import marshmallow as m
+
+from flask_rebar import Rebar
+from flask_rebar import HeaderApiKeyAuthenticator
+from flask_rebar import compat
+from flask_rebar.swagger_generation import SwaggerV2Generator, SwaggerV3Generator
+
+rebar = Rebar()
+registry = rebar.create_handler_registry()
+
+swagger_v2_generator = SwaggerV2Generator()
+swagger_v3_generator = SwaggerV3Generator(include_hidden=True)
```

Comment:
Just checking, but this file is a copy of the registries/legacy.py, correct? With the changes being this include_hidden=True, and hidden=True for /foos and /foo_list ?

Reply:
Correct !